ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/jackson/mixin/ClaimantResponseMixin.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.ccd.jackson.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponseType;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseRejection;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "claimantResponseType")
+@JsonSubTypes({
+    @JsonSubTypes.Type(value = ResponseAcceptation.class, name = "ACCEPTATION"),
+    @JsonSubTypes.Type(value = ResponseRejection.class, name = "REJECTION")
+})
```

Comment:
do we need these? its already available in ClaimantResponse

Reply:
yes .. "property = "claimantResponseType"